clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -6009,6 +6009,8 @@ def note_not_found_by_two_phase_lookup : Note<"%0 should be declared prior to th
 def err_undeclared_use : Error<"use of undeclared %0">;
 def warn_deprecated : Warning<"%0 is deprecated">,
     InGroup<DeprecatedDeclarations>;
+def warn_deprecated_switch_case : Warning<"%0 is deprecated">,
```

Comment:
You can reuse diagnostic text with `warn_deprecated.Summary`. See other examples like:
```
def ext_vla_cxx_in_gnu_mode : Extension<ext_vla_cxx.Summary>,
```

Reply:
Thanks! Done.